src/libraries/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -950,6 +1041,43 @@ void CheckChain()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Android)]
+        public static void BuildInvalidSignatureTwice_Android()
```

Comment:
Like some prior tests, the AIA fetch/etc during this test weren't really part of the test.  If you want to merge the two back together that'd probably be better.